specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/2024-07-26-preview/examples/ledger/PatchRuntimeOptions.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
```

Comment:
Isn't this supposed to have a body with the runtime options that must be patched?